pykern/fconf.py

```diff
@@ -205,7 +206,10 @@ def __init__(self, files):
             except Exception as e:
                 pykern.pkinspect.append_exception_reason(e, f"fconf.Parser.file={f}")
                 raise
-        e = _Evaluator(parser=self)
+        e = _Evaluator(
+            global_fvars=PKDict() if base_vars is None else copy.deepcopy(base_vars),
```

Comment:
A little confused about the `copy.deepcopy(base_vars)`. Why not just `base_vars`?

Reply:
Need to make a copy, because base_vars because the result (global_fvars) of the parser. This means it is destructively modified, which clients of most APIs don't expect.